app/controllers/TemplateController.php

```diff
@@ -1,18 +1,93 @@
 <?php
 
-use Phalcon\Mvc\Model\Resultset;
+use Phalcon\Db;
+use Phalcon\Paginator\Adapter\NativeArray as PaginatorArray;
 
 class TemplateController extends BaseController
 {
     public function listAction()
     {
-        $categoryArray = Category::find(
+        // Fetch URL params
+        $categoryId     = $this->dispatcher->getParam("category");
```

Comment:
Code style. Use single space to separate the operators, fix in all files modified in this pull request.